### DISCUSSION TITLE
Welcome to esc-pos-printer Discussions!

### DISCUSSION THREAD

**Discussion by yayidg22** [yayidg22/esc-pos-printer#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mayerfabian** [comment]
Hello,

Would it be possible to add some additional functions?

I would need to write larger text than just double height or width.
Also, I would need to open the cash desk in certain cases.

Best regards,
Fabian
